content/SCALECLIReference/Network/CLIIPMI.md

```diff
@@ -10,8 +10,158 @@ tags:
+|                 subnet_mask | 255.255.255.0     |
+|  default_gateway_ip_address | 10.200.0.1        |
+| default_gateway_mac_address | 00:00:00:00:00:00 |
+|   backup_gateway_ip_address | 0.0.0.0           |
+|  backup_gateway_mac_address | 00:00:00:00:00:00 |
+|                     vlan_id | 0                 |
+|              vlan_id_enable | false             |
+|               vlan_priority | 0                 |
++-----------------------------+-------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Query Command
+
+The `query` command returns a table of all IPMI channels and their settings.
+
+{{< expand "Running a Simple Query" "v" >}}
+
+#### Description
+The `query` command does not require entering properties or arguments.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of all IPMI instances and their settings, including IP addresses type, IP addresses, MAC addresses, subnet masks, gateway IP addresses, gateway MAC addresses, backup gateway IP addresses, backup gateway MAC addresses, VLAN IDs, VLAN ID statuses, and VLAN ID priorities.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+`network ipmi query`
+
+{{< expand "Command Example" "v" >}}
+```
+network ipmi query
++---------+----+-------------------+-------------+-------------------+---------------+----------------------------+-----------------------------+---------------------------+----------------------------+---------+----------------+---------------+
+| channel | id | ip_address_source | ip_address  | mac_address       | subnet_mask   | default_gateway_ip_address | default_gateway_mac_address | backup_gateway_ip_address | backup_gateway_mac_address | vlan_id | vlan_id_enable | vlan_priority |
++---------+----+-------------------+-------------+-------------------+---------------+----------------------------+-----------------------------+---------------------------+----------------------------+---------+----------------+---------------+
+| 1       | 1  | static            | 10.10.10.10 | 3c:ec:de:5b:5e:70 | 255.255.255.0 | 10.200.0.1                 | 00:00:00:00:00:00           | 0.0.0.0                   | 00:00:00:00:00:00          | 0       | false          | 0             |
++---------+----+-------------------+-------------+-------------------+---------------+----------------------------+-----------------------------+---------------------------+----------------------------+---------+----------------+---------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Update Command
+
+The `update` command allows you to update the settings for a specified IPMI instance.
+
+{{< expand "Updating IPMI Settings" "v" >}}
+
+#### Description
+The `update` command has six configuration properties. They are ipaddress, netmask, gateway, password, dhcp, and vlan.
+You must enter a channel and at least one of the properties for the command to succeed.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
```

Comment:
I believe the text we've standardized on for this type of command reply is "The command returns a blank line."